server/server.h

```diff
@@ -50,6 +50,8 @@ class CStunServerConfig
     
     bool fEnableDosProtection; // enable denial of service protection
 
+    bool fReuseAddr; // if true, the socket option SO_REUSEADDR will be set
```

Comment:
Initialize fReuseAddr to false in construtor.  Look at the very top of server.cpp.  Should be an easy line to add.